src/lib/components/circularprogressbar/CircularProgressBar.component.style.js

```diff
@@ -0,0 +1,35 @@
+import styled from "styled-components";
+import { mergeTheme } from "../../utils";
+import * as defaultTheme from "../../style/theme";
```

Comment:
After the merge the `Improve theme getter` PR,  you may want to update here as well.